changelog/std-all.dd

```diff
@@ -0,0 +1,41 @@
+`std.experimental.all` has been moved to `std`
+
+`std.experimental.all` allowed the convenient use of all Phobos modules
+with one import (`import std.experimental.all`). With this release, this
+convenience module has been stabilized and moved to `std`. From now on, the
+convenience module can be accessed with `import std`:
+
+---
+import std;
+void main()
+{
+    10.iota.map!log.sum.writeln;
+}
+---
+
+For short scripts a lot of imports are often needed to get all the
+modules from the standard library.
```

Comment:
"Scripts and experimental code often use long and frequently changing lists of imports from the standard library."